marklogic/assets/dashboards/overview.json

```diff
@@ -0,0 +1,781 @@
+                "title_size": "16",
+                "type": "toplist"
+            },
+            "id": 4216303646461122,
+            "layout": {
+                "height": 15,
+                "width": 38,
+                "x": 128,
+                "y": 6
+            }
+        },
+        {
+            "definition": {
+                "autoscale": true,
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "last",
+                        "q": "sum:marklogic.databases.forests_count{$database_name}"
+                    }
+                ],
+                "time": {},
+                "title": "Total number of forest by $database_name",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "query_value"
+            },
+            "id": 7180959663064248,
+            "layout": {
+                "height": 8,
+                "width": 15,
+                "x": 131,
+                "y": 22
+            }
+        },
+        {
+            "definition": {
+                "legend_size": "0",
+                "requests": [
+                    {
+                        "display_type": "line",
+                        "q": "sum:marklogic.requests.total_requests{$cluster}",
+                        "style": {
+                            "line_type": "solid",
+                            "line_width": "normal",
+                            "palette": "dog_classic"
+                        }
+                    }
+                ],
+                "show_legend": false,
+                "time": {},
+                "title": "Total number of active requests over $cluster",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "timeseries",
+                "yaxis": {
+                    "include_zero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "id": 6497256550880508,
+            "layout": {
+                "height": 15,
+                "width": 38,
+                "x": 128,
+                "y": 53
+            }
+        },
+        {
+            "definition": {
+                "legend_size": "0",
+                "requests": [
+                    {
+                        "display_type": "line",
+                        "q": "sum:marklogic.forests.total_rate{$forest_name} by {forest_name}",
+                        "style": {
+                            "line_type": "solid",
+                            "line_width": "normal",
+                            "palette": "dog_classic"
+                        }
+                    }
+                ],
+                "show_legend": false,
+                "time": {},
+                "title": "Sum of the processing rate factors by $forest_name",
```

Comment:
There are multiple graphs titled "Sum of the processing rate factors..." in the dashboard, and I don't think it's clear what this metric refers to. The only information I can find in the MarkLogic documentation uses this exact phrase, unfortunately. Is there an alternative wording we can use, or will MarkLogic users know what this means?

Reply:
AFAIU, it's a kind of summary of the queries ands update rates, and I guess MarkLogic users will have a better understanding of what it exactly is.
However, we can remove these graphs and use metrics easier to understand

Reply:
If there aren't more commonly used metrics we can include, I think the "sum of the processing rate factors" graphs should be fine to keep in the dashboard. We've also made it clear that users can clone and customize the dashboard, so this won't shut out any metrics that customers find more meaningful.